config-connector/solutions/iam/kpt/member-iam/Kptfile

```diff
@@ -3,7 +3,8 @@ kind: Kptfile
 metadata:
   name: .
 packageMetadata:
-  shortDescription: Create a service account and grant it permissions to access the desired project.
+  shortDescription: Create a service account and grant it permissions to access the
```

Comment:
This is not a manual change. The latest kpt seems to do auto hard wrap for long text.